superset/views/core.py

```diff
@@ -1437,7 +1437,7 @@ def save_or_overwrite_slice(
         """Save or overwrite a slice"""
         slice_name = args.get('slice_name')
         action = args.get('action')
-        form_data, _ = self.get_form_data()
+        form_data, unused_slc = self.get_form_data()
```

Comment:
oh interesting, I never thought that the gettext standard of using `_` conflicts with the python standard for unused assignments